posts/2021-03-18-async-vision-doc.md

```diff
@@ -0,0 +1,69 @@
+
+### ...and then tells how we will change it
+
+The ultimate goal of the vision doc, of course, is not just to tell us where we are now, but where we are going and how  we will get there. Once we've made good progress on the status quo stories, the next step will be start brainstorming stories about the ["shiny future"][sf].
+
+The idea is to replay the same scenarios from the "status quo" document, except 2 or 3 years in the future. For example, maybe Grace has access to a debugging tool that is able to diagnose her stuck tasks and tell her what kind of future they are blocked on, so she doesn't have to grep through the logs. Maybe the compiler could warn Alan about a likely stack overflow, or (better yet) we can tweak the design of `select` to avoid the problem in the first place. The idea is to be ambitious and focus first and foremost on the user experience we want to create; we'll figure out the steps along the way (and maybe adjust the goal, if we have to).
+
+### Involving the whole community
+
+The async vision document provides a forum where the Async Rust community can plan a great overall experience for Async Rust users. Async Rust was intentionally designed not to have a "one size fits all" mindset, and we don't want to change that. Our goal is to build a shared vision for the end-to-end experience while retaining the loosely coupled, exploration-oriented ecosystem we have built.
+
+The process we are using to write the vision doc encourages active collaboration and "positive sum" thinking. It starts with a brainstorming period, during which we aim to collect as many "status quo" and "shiny future" stories as we can. This brainstorming period runs for six weeks, until the end of April. For the first two weeks (until April 5th), we are collecting "status quo" stories only. After that, we will accept both "status quo" and "shiny future" stories until the end of the brainstorming period. Finally, to cap off the brainstorming period, we will select winners for [awards] like "Most Humorous Story" or "Must Supportive Contributor". 
+
+Once the brainstorming period is complete, the working group leads will begin work on assembling the various stories and shiny futures into a coherent draft. This draft will be reviewed by the community and the Rust teams and adjusted based on feedback.
+
+### Want to help?
+
+If you'd like to help us to write the vision document, we'd love for you to contribute your experiences and vision! Right now, we are focused on creating status quo stories. We are looking for people to author PRs or to talk about their experiences on issues or elsewhere. You can find out more on the [How To Vision][htv] page, which describes the process overall, and especially on the [How To Vision: Status Quo][htvsq] page, which is focused on the "status quo" stories in particular.
```

Comment:
I think explicitly linking to the template status quo doc would be good here - I guess it's not too hard to find, but I think if we want to streamline the process a direct "click this link, type some words, and post" feels like it's going to get us the most value.

Reply:
I did this =)